src/equations/equations.jl

```diff
@@ -181,6 +182,20 @@ end
   return flux
 end
 
+@inline function (boundary_condition::BoundaryConditionDirichlet)(u_inner,
+                                                                  normal_direction::AbstractVector,
+                                                                  x, t, using_nonconservative_flux::True,
+                                                                  nonconservative_surface_flux,
+                                                                  equations)
+  # get the external value of the solution
+  u_boundary = boundary_condition.boundary_value_function(x, t, equations)
+
+  # Calculate boundary flux
+  flux = nonconservative_surface_flux(u_inner, u_boundary, normal_direction, normal_direction, equations)
+
+  return flux
+end
+
```

Comment:
Same here

Reply:
Deleted this based on convo in https://github.com/trixi-framework/Trixi.jl/pull/1431/files#r1192543174